src/emailAlert.ts

```diff
@@ -0,0 +1,35 @@
+const sendgrid = require('@sendgrid/mail')
+
+import { error, log } from './debug'
+
+const api_key = process.env.SENDGRID_API_KEY
+const alert_email = process.env.ALERT_EMAIL
```

Comment:
Also - shouldn't we support multiple destinations? I think that was part of the original idea.